db/migrate/20210809120000_add_svse_attributes.rb

```diff
@@ -0,0 +1,18 @@
+# encoding: utf-8
+
+#  Copyright (c) 2021, Schweizerischer Sportverband öffentlicher Verkehr. This file is part of
+#  hitobito_svse and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito_svse.
+
+class AddSvseAttributes < ActiveRecord::Migration[6.0]
+  def change
+    add_reference :people, :recruited_by, index: false
```

Comment:
Nur für mich, weshalb indexieren wir diese Spalte nicht?

Reply:
Wir haben in hitobito generell keine Foreign Keys. Warum weiss ich auch nicht genau, da müsste uns wohl @amaierhofer oder @kronn erleuchten.

Reply:
Indexieren könnte man die Spalte. `foreign_keys` haben wir vermutlich aus historischen Gründen bisher nicht. Die DB ist bisher eher ein "Dumb Data Store", was früher so der "Rails Way" war. Für MySQL finde ich das noch okay. 

Sobald wir auf PostgreSQL wechseln, könnte man auch gerne direkt einen kurzen Spike hinterherschieben, FKs mal einzubauen. Das Zusammenspiel von `paranioa` und `DELETE ... CASCADE` könnte noch spannend sein. 

Ich wäre grundsätzlich dafür zu haben, etwas mehr Kompetenz in die DB zu schieben. Der erste Schritt ist jedoch für mich ganz klar: eine DB mit Kompetenz in das Projekt einführen. :grin: